### ISSUE TITLE
[Bug] Printing output R summary to console

### ISSUE THREAD

**Issue by annahadji** [wrattler/wrattler#219]
Experienced problems printing the output of R cells to the console.

**Case 1**
Loading in a csv file containing data, and trying to print a table of the counts of unique elements in one of the data's columns,
```
df <- read.csv('data.csv')
print(table(df$column_in_df))
```
results in the following in the console:
```
arrow::Table
```

**Case 2**
Loading in a csv file containing data, and trying to print the summary from an R possion model,
```
install.packages("AER")
library(AER)
df <- read.csv("data.csv")
print(summary(m1 <- glm(crime_count ~ scenic_rating, family="posson", data=df)))
```
results in the following being printed to the console:
```
Received string, but not json format
```
Note that the actual summary can be seen/found using the developer tools.

**Comment by nbarlowATI** [comment]
Case 1 should now be fixed in a908875
 - this was just a namespace issue - arrow::table was masking base::table.

Looking into Case 2 now.

**Comment by nbarlowATI** [comment]
Case 2 should now be fixed in 2f8cd38 - basically the R service was trying to write the glm model to the datastore even though it was NULL after trying to convert it to arrow or json format.
Fixed by putting some more logic into `writeFrame`.

**Comment by annahadji** [comment]
That's perfect thank you, I just double checked and they are now fixed :smiley:
